index.html

```diff
@@ -383,7 +383,6 @@
 	const loadContent = async () => {
 		// Clean URL
 		let path = window.location.pathname + window.location.hash;
-		if (!config.browserRouter && path == "/") path = config.sitePath;
```

Comment:
This breaks the load of the root page, see the test introduced in #39

Also I cannot reproduce your bug. Can you write a test that reproduces it ?